tests/test_nfs.py

```diff
@@ -89,6 +91,17 @@ def test_soft_mount(self, pread, check_server_service, makedirs):
         pread.assert_called_once_with(self.get_soft_mount_pread('mount.nfs',
                                                                 '3'))
 
+    @mock.patch('util.makedirs', autospec=True)
+    @mock.patch('nfs.check_server_service', autospec=True)
+    @mock.patch('util.pread', autospec=True)
+    def test_soft_mount_ipv6(self, pread, check_server_service, makedirs):
+        nfs.soft_mount('mountpoint', 'remoteserver', 'remotepath', 'tcp6',
+                       timeout=None)
+
+        check_server_service.assert_called_once_with('remoteserver')
+        pread.assert_called_once_with(self.get_soft_mount_pread('mount.nfs',
```

Comment:
The scanner doesn't like the duplication of "mount.nfs" but it shouldn't fail the build so happy to leave this as it is without extracting as a constant.